hamocc/mo_netcdf_bgcrw.F90

```diff
@@ -318,8 +320,8 @@ subroutine write_netcdf_var(ncid,desc,arr,klev,time)
       allocate(arr_l(ii,jj,klev))
       allocate(arr_g1(itdm,jj,klev))
 
-      arr_l=0.0
-      arr_g1=0.0
+      arr_l=0.0_rp
+      arr_g1=0.0_rp
```

Comment:
Further down, the routine `nfmpi_put_vara_double_all` is used, so double precision is hard-coded (if PNETCDF is used)